Matrix.java

```diff
@@ -0,0 +1,210 @@
+class Matrix
+{
+    final double[][] a;
+    
+    Matrix(double[][] A)
+    {
+	this(A,0,0);
+    }
+    Matrix(double[][] A, int m, int n)
```

Comment:
We can do this if you want. Just so you know, I don't develop in Eclipse, so mine might be formatted differently (Since I'd have to generate them with a different technique)